code/rdb/datastripe.q

```diff
@@ -2,6 +2,8 @@
 
     .lg.o[`reload;"reload command has been called remotely"];
 
+    t:tables[`.] except .rdb.ignorelist;
```

Comment:
so `t` wasn't defined before?

Reply:
No, this was an oversight as I had assumed it was defined when the initdatastripe function was run